static/app/components/profiling/flamegraphZoomViewMinimap.tsx

```diff
@@ -44,6 +45,7 @@ function FlamegraphZoomViewMinimap({
   >(null);
 
   const dispatch = useDispatchFlamegraphState();
+  const flamegraphSearch = useFlamegraphSearch();
```

Comment:
@JonasBa ; not sure if you have strong opinions on rendering the highlighted frames in the minimap, but i've added it here

Reply:
Meh, I wouldnt. It's too small to see tbh